packages/nimble-components/src/wafer-map/tests/utilities.ts

```diff
@@ -52,7 +52,7 @@ export function getColorScale(): WaferMapColorScale {
     return { colors: ['red', 'blue', 'green'], values: ['1', '2', '3'] };
 }
 
-export function getHighlightedValues(): string[] {
+export function getHighlightedTags(): string[] {
```

Comment:
So, I may be misunderstanding something, but this looks like we are no longer returning tags that will ultimately be highlighted, as none of the dies defined in `getWaferMapDies()` are returning tags with these values.

Reply:
You are right, I added the values from getHighlightedTags() to be among the dies returned by getWaferMapDies(), tests are still passing